specification/frontdoor/resource-manager/Microsoft.Network/stable/2020-08-01/webapplicationfirewall.json

```diff
@@ -1,902 +1,921 @@
-      "properties": {
-        "managedRuleSets": {
-          "description": "List of rule sets.",
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/ManagedRuleSet"
-          }
-        }
-      }
-    },
-    "ManagedRuleSet": {
-      "type": "object",
-      "description": "Defines a managed rule set.",
-      "required": [
-        "ruleSetType",
-        "ruleSetVersion"
-      ],
-      "properties": {
-        "ruleSetType": {
-          "description": "Defines the rule set type to use.",
-          "type": "string"
+    "definitions": {
+        "WebApplicationFirewallPolicy": {
+            "description": "Defines web application firewall policy.",
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "description": "Properties of the web application firewall policy.",
+                    "$ref": "#/definitions/WebApplicationFirewallPolicyProperties"
+                },
+                "etag": {
+                    "type": "string",
+                    "description": "Gets a unique read-only string that changes whenever the resource is updated."
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "./network.json#/definitions/Resource"
+                }
+            ]
+        },
+        "WebApplicationFirewallPolicyProperties": {
+            "description": "Defines web application firewall policy properties.",
+            "properties": {
+                "policySettings": {
+                    "description": "Describes settings for the policy.",
+                    "$ref": "#/definitions/PolicySettings"
+                },
+                "customRules": {
+                    "description": "Describes custom rules inside the policy.",
+                    "$ref": "#/definitions/CustomRuleList"
+                },
+                "managedRules": {
+                    "description": "Describes managed rules inside the policy.",
+                    "$ref": "#/definitions/ManagedRuleSetList"
+                },
+                "frontendEndpointLinks": {
+                    "description": "Describes Frontend Endpoints associated with this Web Application Firewall policy.",
+                    "type": "array",
+                    "readOnly": true,
+                    "items": {
+                        "$ref": "#/definitions/FrontendEndpointLink"
+                    }
+                },
+                "routingRuleLinks": {
+                    "description": "Describes Routing Rules associated with this Web Application Firewall policy.",
+                    "type": "array",
+                    "readOnly": true,
+                    "items": {
+                        "$ref": "#/definitions/RoutingRuleLink"
+                    }
+                },
+                "assignmentPathLinks": {
```

Comment:
This is just a list of frontdoor resource ids, right?

Reply:
According to what we talked today, there would be no way to identify paths from frontend endpoints in the back references. So this should be removed.